source/tasks/DeployTenant/TenantedDeployV6/task.json

```diff
@@ -0,0 +1,126 @@
+﻿{
+    "id": "a847e2d1-5435-4d52-a774-6d300953e85f",
+    "name": "OctopusDeployReleaseTenanted",
+    "friendlyName": "Deploy Octopus Release to Tenants",
+    "description": "Deploy an Octopus Deploy Release to Tenants",
+    "helpMarkDown": "set-by-pack.ps1",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Octopus Deploy",
+    "version": {
+        "Major": 6,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "demands": [],
+    "minimumAgentVersion": "2.144.0",
+    "groups": [
+        {
+            "name": "tenant",
+            "displayName": "Tenants",
+            "isExpanded": true
+        },
+        {
+            "name": "advanced",
+            "displayName": "Advanced Options",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "OctoConnectedServiceName",
+            "type": "connectedService:OctopusEndpoint",
+            "label": "Octopus Deploy Server",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Octopus Deploy server connection"
+        },
+        {
+            "name": "Space",
+            "type": "string",
+            "label": "Space",
+            "defaultValue": "",
+            "required": true,
+            "properties": {
+                "EditableOptions": "True"
```

Comment:
this isn't needed now we aren't using the selectors

Reply:
fixed